src/main/kotlin/nl/info/zac/app/planitems/PlanItemsRestService.kt

```diff
@@ -158,12 +159,14 @@ class PlanItemsRestService @Inject constructor(
             }
         }
 
-        if (humanTaskData.taakStuurGegevens.sendMail && humanTaskData.taakStuurGegevens.mail != null) {
-            val mail = Mail.valueOf(humanTaskData.taakStuurGegevens.mail as String)
+        val sendMail = TaakVariabelenService.isSendDataSendMail(taakdata) || humanTaskData.taakStuurGegevens?.sendMail ?: false
+        val mail = TaakVariabelenService.readSendDataMail(taakdata).getOrNull() ?: humanTaskData.taakStuurGegevens?.mail
```

Comment:
Multiple vars named `mail` is a bit confusing. Can this one not be renamed maybe?

Reply:
Renamed to `sendDataMail`